spec/unit/recipes/default_spec.rb

```diff
@@ -83,6 +83,8 @@
         )
       end
 
+      it { expect(chef_run).to install_cron_package('osl-docker') }
```

Comment:
Please also add the test for enabling/starting the service.

Reply:
Pushed a change to check if the service is enabled/started